pynamodb/attributes.py

```diff
@@ -253,6 +254,17 @@ def serialize(self, value):
     def deserialize(self, value):
         return bool(json.loads(value))
 
+    def get_value(self, value):
```

Comment:
Why is this logic on the generic `Attribute`? I would've expected it to be on the more specific Boolean subclasses